fc/code_generation.py

```diff
@@ -220,7 +218,7 @@ def derivative_name(deriv):
 
         # Allow special output value 'state_variable'
         # TODO Replace this with a more generic implementation
-        if output == 'state_variable':
+        if output[1] == 'state_variable':
```

Comment:
I didn't realise `state_variable` had a URI?

Reply:
Yep, it's a standard oxmeta:state_variable reference.